virtual/python-typing/python-typing-0.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} pypy{,3} )
```

Comment:
Correct me if I'm wrong but since the `typing` module is shipping with Python 3.5 and 3.6, these versions shouldn't be listed here, should they?

Reply:
They should. This allows a package, which supports e.g. python2.7 and python3.5, depend on virtual-typing. Otherwise it would be necessary to create a conditional dep for pythons < 3.5, which defies the purpose of this virtual. Check out virtual/python-enum34.